### PR TITLE
[INTERNAL] Consolidate UI5 Tooling Documentation part II

### PR DESCRIPTION
**Pull request by RandomByte**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -7,8 +7,14 @@ site_dir: 'site' # default
 nav:
     - Home: index.md
     - Getting Started: pages/GettingStarted.md
+    - Working with the UI5 Tooling:
```
Comment (matz3) [inline mkdocs.yml]: That's quite a long title..

Comment (RandomByte) [reply mkdocs.yml]: Yes

Comment (RandomByte) [reply mkdocs.yml]: Can we live with it for the time being?

Comment (RandomByte) [reply mkdocs.yml]: Renamed to "Workflows"

**Pull request merged**